website/src/roadmap/euphoria.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: section
+title: "Euphoria API Roadmap"
+permalink: /roadmap/euphoria/
+section_menu: section-menu/roadmap.html
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Euphoria API
```

Comment:
@dmvk @VaclavPlajt See how I have copied text from elsewhere here. If there is more you would like to say, is it OK to do it in a follow-up PR? Or issue a PR to my branch.

http://apache-beam-website-pull-requests.storage.googleapis.com/6718/roadmap/euphoria/index.html

Reply:
@kennknowles this is great 👍 We'll fill in the roadmap from our side within next week, thanks.